lib/plug/parsers.ex

```diff
@@ -206,19 +207,19 @@ defmodule Plug.Parsers do
     merge_params(conn, make_empty_if_unfetched(body_params))
   end
 
-  defp reduce(conn, [h|t], type, subtype, headers, opts) do
-    case h.parse(conn, type, subtype, headers, opts) do
+  defp reduce(conn, [h|t], content_type, opts) do
```

Comment:
Since we're touching this line, it perhaps makes sense to do `[h|t]` –> `[parser | rest]` renaming (and add spaces around `|`).